### PR TITLE
RemovingGoogleCollection

### PR DESCRIPTION
**Pull request by laarchenko**

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/appnexus/AppnexusBidder.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import com.google.common.collect.Lists;
+import org.apache.commons.collections4.ListUtils;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/bidder/appnexus/AppnexusBidder.java:6]: Import ordering.

File: src/main/java/org/prebid/server/bidder/bidscube/BidscubeBidder.java [hunk 2]
```diff
@@ -22,20 +21,23 @@
 import org.prebid.server.proto.openrtb.ext.response.BidType;
 import org.prebid.server.util.HttpUtil;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
 import java.util.Objects;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/bidder/bidscube/BidscubeBidder.java:24]: Import ordering.

File: src/main/java/org/prebid/server/bidder/madvertise/MadvertiseBidder.java [hunk 3]
```diff
@@ -25,11 +24,13 @@
 import org.prebid.server.proto.openrtb.ext.response.BidType;
 import org.prebid.server.util.HttpUtil;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
 import java.util.Objects;
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/bidder/madvertise/MadvertiseBidder.java:27]: Import order.

**Review by Net-burst — CHANGES_REQUESTED** [review]

**Review by rpanchyk — APPROVED** [review]

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**